CHANGELOG.md

```diff
@@ -6,6 +6,19 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## [Unreleased]
 
+## [0.23.0] - 2020-01-31
+
+### Added
+
+- ([GH-213](https://github.com/lingua-pupuli/puppet-editor-services/issues/213)) [puppet-editor-services-0.24.0](https://github.com/lingua-pupuli/puppet-editor-services/releases/tag/0.24.0) Gather facts using sidecar
```

Comment:
Can probably squish this into a single entry.  Also this is a private implementation detail so no user impact (I hope! :-) )

Reply:
Merged into one entry